conf/messages.en

```diff
@@ -97,11 +98,11 @@ penalisedEstateAgentsActDetail.error.length = Description of the penalty must be
 penalisedEstateAgentsActDetail.error.punctuation = Description of the penalty can include letters a to z, numbers, spaces and punctuation
 
 penalisedProfessionalBody.title = Has the business, or anyone in the business, ever been penalised by a professional body for estate agency business activities?
-penalisedProfessionalBody.heading = Has the business, or anyone in the business, ever been penalised by a professional body for estate agency business activities?
+penalisedProfessionalBody.heading = Penalties by a professional body
```

Comment:
I think because the h1 has changed, the page title should also change to match. I can't see mention of it on the headings design system page but I feel like I have vague memory of this being flagged in a11y reviews before.

Reply:
Updated :)